eras/byron/crypto/test/cardano-crypto-test.cabal

```diff
@@ -51,7 +51,7 @@ library
         -Wno-safe -Wno-unsafe -Wunused-packages
 
     build-depends:
-        base >=4.14 && <4.19,
+        base >=4.14 && <5,
```

Comment:
Change like this requires a patch version update. Could you please bump up the version of `cardano-crypto-test` in the cabal file from `1.5.0.0` to `1.5.0.1`